.gitignore

```diff
@@ -39,4 +43,4 @@ source/state_representation/build
 # gtest lib
 source/dynamical_systems/lib/googletest
 source/robot_model/lib/googletest
```

Comment:
do we still need all these 3 lines?

Reply:
no - i was thinking about removing these but I thought it will have to happen anyway in the next PRs. I guess there's still scope for it in this one!